eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/EnactSpec.hs

```diff
@@ -47,11 +50,18 @@ spec ::
   SpecWith (ImpTestState era)
 spec =
   describe "ENACT" $ do
+    relevantDuringBootstrapSpec
     treasuryWithdrawalsSpec
-    hardForkInitiationSpec
     noConfidenceSpec
     constitutionSpec
-    actionPrioritySpec
+    actionPriorityCommitteePurposeSpec
+    hardForkInitiationSpec
+
+relevantDuringBootstrapSpec ::
+  ConwayEraImp era =>
+  SpecWith (ImpTestState era)
+relevantDuringBootstrapSpec = do
+  actionPrioritySpec
```

Comment:
We do need some tests that check `hardForkInitiation` during bootstrap. It is in fact more relevant for bootstrap, rather than post boostrap at this point

Reply:
Good point! I added a test with only SPOs voting  to the bootstrap section of this test.